pkg/controller/servicebindingrequest/controller.go

```diff
@@ -42,6 +42,21 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	pred := predicate.Funcs{
 		UpdateFunc: func(e event.UpdateEvent) bool {
+
+			if e.ObjectOld.DeepCopyObject().GetObjectKind().GroupVersionKind().Kind == "ServiceBindingRequest" {
```

Comment:
@sbose78 you've created `ServiceBindingRequestKind` const variable, why not use that instead of `"ServiceBindingRequest"` string here.

Reply:
That was a miss, thanks! fixed ba34d7f